content/vault/v1.18.x/content/docs/upgrading/upgrade-to-1.18.x.mdx

```diff
@@ -92,6 +92,14 @@ WARNING! The following warnings were returned from Vault:
 
 </CodeBlockConfig>
 
+#### Counters API requires start time in OSS Vault
+
+The `/sys/internal/counters/activity` API requires the `start_time` parameter when used 
+with OSS Vault. Unlike Enterprise Vault, OSS Vault does not define a billing start time, and ommitting `start_time` may lead
+to unexpected behavior in affected versions (1.18.x and 1.19.x). In Enterprise Vault, `start_time` parameter is optional and 
+can be safely ommitted, as it defaults to the cluster's billing start date. To avoid issues in OSS Vault, ensure that your 
+API calls explicitely provide a valid `start_time`.
```

Comment:
```suggestion
Vault Community requires the `start_time` parameter  for
`/sys/internal/counters/activity`. Unlike Enterprise Vault, Vault Commuity does
not define a billing start time, and ommitting `start_time` may lead to
unexpected behavior in affected versions (1.18.x and 1.19.x). 

You can safely omit `start_time` for Vault Enterprise as `start_time` defaults
to the billing start date for the cluster. To avoid issues and unexpected
behavior, always provide an explicit start time if you use Vault Community.
```

Style correction: use proper names for Vault editions (Vault Enterprise, Vault Community), write in active voice